index.js

```diff
@@ -11,6 +11,7 @@ module.exports = function (movieString) {
     movie = baseName;
   }
 
-  return movie;
+  movie = movie.substring(movie.indexOf('-') + 1);
```

Comment:
@noamokman, @danielhusar: This line seems like it might cut off too much, e.g. in the tests

    assert.equal(movieTitle('snow-white.bird.in.a.blizzard.2014.limited.720p.bluray.x264.mkv'), 'white bird in a blizzard');

should really be _snow-white bird in a blizzard_